src/Components/Navbar/Navbar.jsx

```diff
@@ -1,6 +1,121 @@
+import { useState, useEffect, useRef } from "react";
+import { NavLink } from "react-router-dom";
 import style from "./Navbar.module.scss";
+
 const Navbar = () => {
-  return <nav className={style.navbar}>Navbar</nav>;
+  const [burger, setBurger] = useState(false);
```

Comment:
ambiguous variable name i.e. name does not provide context regarding the purpose of the variable.